messaging/src/main/java/org/jboss/as/messaging/jms/bridge/JMSBridgeService.java

```diff
@@ -93,18 +93,20 @@ public void run() {
     }
 
     public void startBridge() throws Exception {
-        if (moduleName == null) {
-            bridge.start();
+        final Module module;
+        if (moduleName != null) {
+            ModuleIdentifier moduleID = ModuleIdentifier.create(moduleName);
+            module = Module.getCallerModuleLoader().loadModule(moduleID);
```

Comment:
I've updated the code to remove the call to `getCallerModuler` and retrieves instead the context module loader